bugshaker/src/main/java/com/github/stkent/bugshaker/flow/email/FeedbackEmailFlowManager.java

```diff
@@ -59,17 +61,29 @@
 	private boolean ignoreFlagSecure;
 	private ScreenshotProvider screenshotProvider;
 	private ActivityReferenceManager activityReferenceManager;
+	private Application app;
 
 	public FeedbackEmailFlowManager(ScreenshotProvider screenshotProvider, Application application) {
 		this.screenshotProvider = ScreenshotUtil.getScreenshotProvider(application);
 		activityReferenceManager = new ActivityReferenceManager();
+		app = application;
```

Comment:
We don't need the instance of the application here. We can just store the context!